prow/mason_lib.sh

```diff
@@ -48,6 +48,8 @@ function get_resource() {
     grep -q READY ${file_log} && ready=true || ready=false
     if [[ ${ready} == true ]]; then
       cat "${info_path}"
+      local project="$(cat "${info_path}" | head -n 1 | tr -d ':')"
```

Comment:
I know I did that but could please you change ```cat "${info_path}" | head -n 1``` to ```head -n 1 "${info_path}"```

Reply:
sure